### DISCUSSION TITLE
Bruns engine encryption possibility?

### DISCUSSION THREAD

**Discussion by HHCThuan** [YuriSizuku/GalgameReverse#13] [General]
This might not be the place, but since bruns engine is so obscure, I'm firing my shot anyway.

So the decryption works, and the dialogues can be translated that way, but now comes the problem of encrypting the txt file back into bso.

I've looked into the libscr.dll and there seems to be no opposite counterpart for the file_to_bytearray_nosearch function, which is used for the decryption. And since it's a dll, you'd need to reverse engineer it to learn how it works. This would take a long time.

So I'd like to know if there's any less hard way to encrypt it back into a bso file, so that I can translate some Bruns engine games, thank you so much.

**Comment by YuriSizuku** [comment]
You don't need to encrypt the text. This engine can directly load decrypted files.

**Reply by HHCThuan** [reply]
Huh... well, thanks, It works, I can't really believe it. I thought they went through all the trouble of encrypting game files is so that we can't mess with it. Thank you so much, now I can get started the translation, this was a life saver.
